### ISSUE TITLE
[Bug Report] el-date-picker  当type为datetime时候，format='yyyy/MM/dd HH时'   对于value的值的匹配有问题

### ISSUE THREAD

**Issue by wangenze267** [ElemeFE/element#22490]
### Element UI version
2.15.13

### OS/Browsers version
edge 112.0.1722.46

### Vue version
2.7.14

### Reproduction Link
https://codepen.io/wangenze267/pen/ExdPqPW

### Steps to reproduce
https://codepen.io/wangenze267/pen/ExdPqPW
 点击选择一个日期，携带时分秒
弹出层上方日期输入框中，也携带了  format='yyyy/MM/dd HH时'  中的时   它本不应该出现

或者使用如下代码
`<el-date-picker v-model="value" type="datetime" value-format="yyyy-MM-dd HH:mm:ss" format="yyyy/MM/dd HH时" placeholder="选择日期时间"> </el-date-picker>`

### What is Expected?
弹出层中日期的输入框中不应出现格式化数值与时分秒相关的字符

### What is actually happening?
目前是出现的

**Comment by webvs2** [comment]
不需要小时的话，为什么不选择DatePicker 呢

**Comment by wangenze267** [comment]
需要小时呀，但是你看我箭头的地方 左侧是日期输入框中有了不该出现的单位 “时”

**Comment by webvs2** [comment]
你的问题是两个输入框都被可视化了是吗

**Comment by wangenze267** [comment]
不是 我的问题是 日期的那个输入框中出现了 "时"   按照我的format    我只是在时分秒后增加了单位 "时" ，在弹出框中，时分秒的输入框中出现  “时”  ，我觉得是正常的   但是 弹出框的日期输入框中却出现了本不应该出现的单位 “时”

**Comment by webvs2** [comment]
明白了，不过这是否要限制需要考虑下，“时”对我们来说是语义化的显示，对程序来说只是匹配，这将处理多种“时”的语义化。

> 不是 我的问题是 日期的那个输入框中出现了 "时" 按照我的format 我只是在时分秒后增加了单位 "时" ，在弹出框中，时分秒的输入框中出现 “时” ，我觉得是正常的 但是 弹出框的日期输入框中却出现了本不应该出现的单位 “时”

**Comment by wangenze267** [comment]
是这样的，我看了一下源码中的这部分，修改起来是要考虑很多方面的，但是对于使用者来说，这仍是个问题

**Comment by webvs2** [comment]
> 是这样的，我看了一下源码中的这部分，修改起来是要考虑很多方面的，但是对于使用者来说，这仍是个问题

将尝试组件内部的处理来锁死这种边缘情况。👍感谢发现问题

**Comment by wangenze267** [comment]
辛苦！！！
